### PR TITLE
Disable next button on Python wizard creation

### PR DESCRIPTION
**Pull request by lucyzhang929**
This PR adds a check in the wizard navigation validator to return false when the pages have not been registered yet. Previously, a user can click next before the installation path is loaded on the first page and the wizard ends up in this weird state where it goes to the second page but the kernel dependencies never load and the previous button does not work. 

Screen Shot 2022-01-13 at 10 00 23 AM

I think this is what is happening during the Python smoke test that fails occasionally. Although it is still not clear what is causing the next button to be clicked (logs don’t show any clicks or enter key pressed) - I will continue to investigate that.

### CODE REVIEW

**Review by Charles-Gagnon — APPROVED** [review]

File: extensions/notebook/src/dialog/configurePython/configurePythonWizard.ts [hunk 1]
```diff
@@ -119,6 +119,7 @@ export class ConfigurePythonWizard {
 			return true;
 		});
 
+		this._wizard.nextButton.enabled = false;
```
Comment (chlafreniere) [inline extensions/notebook/src/dialog/configurePython/configurePythonWizard.ts:122]: For this particular scenario, any thoughts on if we should log when the next button is enabled, and when the rest of the page has loaded (just for sanity)?

**Comment by corivera** [comment]
Is the page navigation validation not working here? We call onPageLeave() when trying to move to a new page, and if that method returns false we stay on the current page. Or at least that's how it should be working. The path configuration page's onPageLeave() returns false if the loading spinner is still active, for example.

**Review by corivera — CHANGES_REQUESTED** [review]
See my other comment about page validation.

**Comment by corivera** [comment]
Here's the line where we check the page validation: https://github.com/microsoft/azuredatastudio/blob/main/extensions/notebook/src/dialog/configurePython/configurePythonWizard.ts#L112

**Comment by lucyzhang929** [comment]
@corivera The issue is when next is clicked too quickly and the pages have not been registered yet, the navigation validator returns true. Actually a better fix would be to add a check in the navigation validator to return false if pages have not been loaded yet. Let me update the PR.

**Review by corivera — APPROVED** [review]

**Pull request merged**